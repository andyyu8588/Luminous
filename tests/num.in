print(123 / 3 * 5 + 7 - 9 * 2);
print((3 - 1) / (3 - 1) * (9 + 6) * (2 * 2));
print(-1 - 1);
print(-2 * -2);
print(-3 / 2);
print(-3.1415926 + 3.1415926);
print(35 / -4);