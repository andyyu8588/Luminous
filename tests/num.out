4
60
-2
4
-1.5
0
-8.75
